README.md

```diff
@@ -336,58 +336,3 @@ alerts.yml  metrics.yml  node-density.yml  pod.yml  metrics-report.yml
 $ vi node-density.yml                               # Perform modifications accordingly
 $ kube-burner-ocp node-density --pods-per-node=100  # Run workload
 ```
-
-## Cluster metadata
```

Comment:
are we able to keep some doucmentation around this type of data that gets printed off?

Reply:
@rsevilla87 looks like the documentation commit you added somehow got missed in this[ PR merge](https://github.com/kube-burner/kube-burner/pull/656). We will need to update it in our upcoming release.